examples/deepspeed/deepspeed_dcgan/cifar10_zero2.yaml

```diff
@@ -18,7 +18,7 @@ environment:
         # You may need to modify this to match your network configuration.                          
         - NCCL_SOCKET_IFNAME=ens,eth,ib
     image:
-      gpu: determinedai/environments:cuda-11.3-pytorch-1.10-deepspeed-0.8.3-gpu-0.21.2
```

Comment:
same comment here on image versions.  It's dangerous to do these in a large PR, because you really need to be sure that there aren't any bumpenv efforts in flight or you can end up with your image "left behind".

If you want to just confirm this is up-to-date, I'm fine with that.  I'll probably approve this either tonight or early tomorrow, I expect.

Reply:
Is this addressed by #6988? Wasn't entirely sure what you meant. Replaced all `determinedai/environments:cuda-11.3-pytorch-1.10-deepspeed-0.8.3-gpu-6eceaca` instances with `determinedai/environments:cuda-11.3-pytorch-1.10-deepspeed-0.8.3-gpu-0.22.1`

Reply:
https://github.com/determined-ai/determined/blob/main/tools/scripts/bumpenvs.yaml#L13-L16

I didn't initially understand bumpenvs, this is why we have this desync. To the best of my understanding, when bumpenvs is run, we update this "new" column to "old" and change all the names. Since both the 0.21.2 version of the environment and the 6eceaca version under new, this means that when we run bumpenvs next it will update them all correctly right? I think I'd prefer to just choose one or the other, but was settling on the 6eceaca version for clarity, is that okay?